content/_includes/logos/oddcontrast.svg

```diff
@@ -0,0 +1,27 @@
+<svg
+  data-logo="oddcontrast"
+  xmlns="http://www.w3.org/2000/svg"
+  aria-hidden="true"
+  focusable="false"
+  width="511.81px"
+  height="63.45px"
```

Comment:
@stacyk I made up these height + width values -- not sure if you want to adjust?

Reply:
@jgerigmeyer They look fine to me. As long as it matches the aspect ratio in the viewbox (or exactly) then I think it is fine. We don't need the px unit there, but it doesn't hurt anything.